rest/accounts/list-get-example-2/list-get-example-2.4.x.rb

```diff
@@ -10,6 +10,6 @@
 
 # Loop over accounts and print out a property for each one
 @client.accounts.list({
-    :status => "active"}).each do |account|
-    puts account.friendly_name
+  :status => "active"}).each do |account|
```

Comment:
No need for curly braces in this example. Please take a look at https://github.com/bbatsov/ruby-style-guide#no-braces-opts-hash